pkg/gofr/cmd/terminal/colors.go

```diff
@@ -0,0 +1,30 @@
+package terminal
+
+import "strconv"
+
+const (
+	Black = iota
+	Red
+	Green
+	Yellow
+	Blue
+	Magenta
+	Cyan
+	White
+	BrightBlack
+	BrightRed
+	BrightGreen
+	BrightYellow
+	BrightBlue
+	BrightMagenta
+	BrightCyan
+	BrightWhite
+)
+
+func (o *Output) SetColor(colorCode int) {
+	o.Printf(csi + "38;5;" + strconv.Itoa(colorCode) + "m")
```

Comment:
Why a sprintf if you concatenating string, and do not use %s or %d

It would avoid the call to strings.Itoa